src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
+* {
+  box-sizing: border-box;
 }
+
+$cell-size: 100px;
+$gap: 1px;
+
+@import 'body';
+@import 'calendar';
+@import 'functions';
```

Comment:
When importing SCSS files, it's a good practice to omit the file extension. Also, ensure that the order of imports follows the logical structure of the project, with the most general styles (like 'body') being imported before more specific ones (like 'calendar').